truffle-config-b.js

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Why do we have this file, if we can use the already existing `truffle-config.js` file?

Reply:
completely agree